### DISCUSSION TITLE
Custom font for relatedRecordsList

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16682] [How to]
I managed to use custom fonts in pdf templates but cannot figure out how to use it with relatedRecordsList or other vars.
What would be the optimal approach?

**Comment by rskrzypczak** [comment]
Try to use twig and create your own table: https://github.com/YetiForceCompany/YetiForceCRM/discussions/16588#discussioncomment-3892560
